LayoutTests/fast/html/details-no-summary-keyboard-show-hide.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
Can we stop using the pre/post.js versions?

Reply:
Sure.